ui/page/main_page.go

```diff
@@ -470,6 +482,24 @@ func (mp *MainPage) LayoutUSDBalance(gtx layout.Context) layout.Dimensions {
 	)
 }
 
+func (mp *MainPage) hideMyBalance(gtx layout.Context) layout.Dimensions {
```

Comment:
```suggestion
func (mp *MainPage) totalDCRBalance(gtx layout.Context) layout.Dimensions {
```

This is a more appropriate function name